dockerise.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
best practise isn't to use a shell script, it's fine for now and for testing but you're better off building a k8s environment that is deployed using ansible.

the setup steps done below shouldn't be done using bash but ansible instead.

The reason for using ansible over bash is because it allows for serverless deployments which in the long run makes life a lot cheaper, quicker and easier